.gitignore

```diff
@@ -13,3 +13,4 @@ coverage
 globalConfig.json
 .envrc
 tsconfig.tsbuildinfo
+ecosystem.config.js
```

Comment:
What's this one?

Reply:
its the setup file for a system i use to run the api locally